models/biocro/R/run.biocro.R

```diff
@@ -2,56 +2,23 @@
 #'
 #' @param lat latitude in decimal degrees
 #' @param lon longitude in decimal degrees
-#' @param metfile full path and name of a netCDF file in either a PEcAn-CF format with meteorological driver data
-#' or a csv file with hourly data in BioCro format
+#' @param metpath full path and name prefix of a csv file with hourly data in BioCro format,
+#' e.g. `/dir/met` if the files to be used are `/dir/met.2004.csv` and `dir/met.2005.csv'
 #' @param soil.nc full path and name of a netCDF file with soil data
 #' @param config full path and name of a config.xml file containing parameter values and configuration information for BioCro
 #' @param coppice.interval numeric, number of years between cuttings for coppice plant or perinneal grass (default 1)
 #' @return output from one of the \code{BioCro::*.Gro} functions (determined by \code{config$genus}), as data.table object
 #' @export
 #' @author David LeBauer
-run.biocro <- function(lat, lon, metfile, soil.nc = NULL, config = config, coppice.interval = 1, 
+run.biocro <- function(lat, lon, metpath, soil.nc = NULL, config = config, coppice.interval = 1,
```

Comment:
you can also remove the `met.uncertainty` and `irrigation` arguments from this. Would probably be better to either a) add a new variable in the met files like irrigation_flux that can be added to precipitation_flux explicitly inside the code or have a separate irrigation file. But that is not currently a priority. Implementation of a scheme for met (and other driver) uncertainty is discussed in #648